packages/stripe-sdk/src/stripe-sdk.ts

```diff
@@ -0,0 +1,19 @@
+import {Context, defaultContext} from './stripe-client-context'
+
+type SDKOptions = {ctx?: Context}
+
+export function getStripeSdk(
+  {ctx = defaultContext}: SDKOptions = {ctx: defaultContext},
+) {
+  return {
+    async getCheckoutSession(checkoutSessionId: string) {
+      return await ctx.stripe.checkout.sessions.retrieve(checkoutSessionId, {
+        expand: [
+          'customer',
+          'line_items.data.price.product',
+          'payment_intent.charges',
+        ],
+      })
+    },
+  }
+}
```

Comment:
by default the context carries the normal `stripe` client instance, but we can also mock it

not I didn't write a test and exercise the mock context to see if it works as expected, but hand rolling a mock is also simple and can **iterate into end game automated solution** as needed (versus aiming for that now)